setup.py

```diff
@@ -3,7 +3,7 @@
 
 
 install_requires = [
-    "beautifulsoup4>=4.5.0,<4.7",
+    "beautifulsoup4>=4.5.0,<4.9",
     "Django>=1.11,<2.3",
     "dj-database-url>=0.4.2,<1",
     "python-dateutil>=2.1<3",
```

Comment:
This doesn't involve your PR, but since we're here, we could fix syntax:

```suggestion
    "python-dateutil>=2.1,<3",
```

Reply:
Good catch Bill, and I am glad that the changes work for you.
I will fix the syntax for python-dateutil and merge this PR into master